### DISCUSSION TITLE
外置登录authlib injector相关，应该跟PCL没啥关系（？）

### DISCUSSION THREAD

**Discussion by WratithCr** [Meloong-Git/PCL#1406] [提问] [answered]
使用Authlib-injector方式启动会不定时抽风，表现为用此方式登入的服务器会不定期不定时随机爆破（连接中断失去信号），此时退出游戏重新启动会提示网络问题卡在42.26%，然后提示网络问题启动失败，但是挂上VPN登入游戏成功之后已然无法与服务器建立连接，我不是很清楚该方式出现该类天坑的原因，但是以我的经历而言频率相当之高，一周起码碰见十几次，我觉得猫猫桑有必要了解一下，可以的话加入一个公告在启动失败的时候弹出来说明一下，三天两头抽风是真的挺让人上火的。

**Comment by WForst-Breeze** [comment, marked as answer]
hmm……可以提交一个[优化建议](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E4%BC%98%E5%8C%96%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=ch.yml)，谢谢~
